### ISSUE TITLE
Create a basic Alaveteli install for Liberia 

### ISSUE THREAD

**Issue by JenMysoc** [mysociety/alaveteli#2237]
This install is to allow testing in Liberia by users to get feedback from them on how it might fit into the ecosystem and project

#### Todo

- [x] Need a contact email for the `team@` address to forward to.
- [x] Configure outgoing emails to go to `team@` address

**Comment by garethrees** [comment]
Need a contact email for the team@ address to forward to.

**Comment by garethrees** [comment]
Pretty much done; just waiting on DNS to give it a full test

**Comment by garethrees** [comment]
Completed request process with outgoing and incoming email http://liberia.alaveteli.org/request/test_2

**Comment by garethrees** [comment]
>  Need a contact email for the team@ address to forward to.

This has been changed to @JenMysoc's email until they provide one. Easiest to just submit a new ticket when they send it over.
